### PR TITLE
sentinel variable-Add feature

### PR DESCRIPTION
**Pull request by bernicuts**
cod ready for first 2 task

### CODE REVIEW

File: src/unit_convert.java [hunk 1]
```diff
@@ -5,19 +5,20 @@
 	public static void main (String[] args) {
 		System.out.println("___the conversion is being started___");
 		System.out.println("-\\hai sa comparam mere cu pere\\-");
-
-		Scanner user_int = new Scanner(System.in);
 		
-		boolean varType = true ;
+		boolean varType = true ; // that would be a sentinel variable
```
Comment (bernicuts) [inline src/unit_convert.java:9]: sentiner variable

File: src/unit_convert.java [hunk 2]
```diff
@@ -5,19 +5,20 @@
...
+				Scanner user_int = new Scanner(System.in);
 				System.out.print("provide your integer:");
 				int numar = user_int.nextInt();
-				System.out.print("you have entered:" + numar);
+				System.out.print("you have successfuly entered:" + numar);
 				varType = false;
 			}
 			catch (Exception e) {
-				System.out.print("value invalid");
+				System.out.println("value invalid, try again with a number...\n");
```
Comment (bernicuts) [inline src/unit_convert.java:21]: tre de pus inca un punct

File: src/unit_convert.java [hunk 3]
```diff
@@ -5,19 +5,20 @@
 	public static void main (String[] args) {
```
Comment (bernicuts) [inline src/unit_convert.java:5]: ce face asta

File: src/unit_convert.java [hunk 4]
```diff
@@ -5,19 +5,20 @@
...
 		System.out.println("___the conversion is being started___");
 		System.out.println("-\\hai sa comparam mere cu pere\\-");
```
Comment (bernicuts) [inline src/unit_convert.java:7]: aduce la ecran

File: .classpath [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (marcelbernic) [inline .classpath:1]: this file is generated by eclispe. Normaly this file should exist ONLY on your local repo (on you computer). Think about it, maybe I don't use eclipse (I use IntelliJ IDE) so this file is not pertinent for me (or other developer). For this reason there is a '.gitignore' file  in you git repo. In this file we specify wich files we don't add to push on remote repo (files that we want that git IGNORES). You can specify also a file pattern in git ignore. Read a little bit about gitignore and his purpose. And add this file to gitignore so we don't track it on remote repo. On the web there exists tools to generate some gitignore file (https://www.gitignore.io/) , feel free to explore. You can specify here: java, eclipse, intellij and the tool we'll generate for this file. Still, at this stage is better to write your own .gitingnore file so you better understand. To complete this comment add this file(.classpath) to .gitignore.

File: .project [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (marcelbernic) [inline .project:1]: Same comment as for the file .casspath

File: src/unit_convert.java [hunk 7]
```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
 
+//adaugat metric-converter
```
Comment (marcelbernic) [inline src/unit_convert.java:3]: Remove this comment

File: src/unit_convert.java [hunk 8]
```diff
@@ -1,10 +1,30 @@
...
 public class unit_convert {
 	public static void main (String[] args) {
-		System.out.println("the conversion is being started");
-		System.out.println("hai sa comparam mere cu pere");
+		System.out.println("___the conversion is being started___");
```
Comment (marcelbernic) [inline src/unit_convert.java:6]: Chose a better (significant) message for the user

File: src/unit_convert.java [hunk 9]
```diff
@@ -1,10 +1,30 @@
...
+		System.out.println("-\\hai sa comparam mere cu pere\\-");
```
Comment (marcelbernic) [inline src/unit_convert.java:7]: Remove this comment (zero effect on the code and on the user)

File: src/unit_convert.java [hunk 10]
```diff
@@ -1,10 +1,30 @@
...
+		
+		boolean varType = true ; // that would be a sentinel variable
```
Comment (marcelbernic) [inline src/unit_convert.java:9]: Chose a better name for this variable (significant, meaningful)

File: src/unit_convert.java [hunk 11]
```diff
@@ -1,10 +1,30 @@
...
+		
+		while(varType) {
+
```
Comment (marcelbernic) [inline src/unit_convert.java:12]: remove this empty line

File: src/unit_convert.java [hunk 12]
```diff
@@ -1,10 +1,30 @@
...
+		while(varType) {
+			
+			try {
+				Scanner user_int = new Scanner(System.in);
+				System.out.print("provide your integer:");
+				int numar = user_int.nextInt();
+				System.out.print("you have successfuly entered:" + numar);
+				varType = false;
+			}
+			catch (Exception e) {
```
Comment (marcelbernic) [inline src/unit_convert.java:20]: Here you catch ANY exception that is thrown in your try block. Please catch only the exception throwed by 'user_int.nextInt()'. We don't want to generalize a specific problem (that's not a good practice). Ask @marcelbernic  if this comment is not clear.

File: src/unit_convert.java [hunk 13]
```diff
@@ -1,10 +1,30 @@
...
+				System.out.println("value invalid, try again with a number...\n");
```
Comment (marcelbernic) [inline src/unit_convert.java:21]: Write a better message (think that you are the user... -200 is a number, but maybe this number is not a valid number for our application).

**Review by marcelbernic — CHANGES_REQUESTED** [review]
Fix everything specified in the comments (reach @marcelbernic if you have any questions). Once everything is fixed we'll merge the code in master (the master is your PROD, we're gonna put this feature in PROD).

File: .classpath (hunk 5 above)
Comment (bernicuts) [reply .classpath:1]: added git .ignore with content generated on www.gitignore.io

**Review by bernicuts — COMMENTED** [review]
all is fixed

**Pull request merged**